deliver/lib/deliver/app_screenshot_iterator.rb

```diff
@@ -0,0 +1,98 @@
+module Deliver
+  # This is a convinient class that enumerates app store connect's screenshots in various degrees.
+  class AppScreenshotIterator
+    NUMBER_OF_THREADS = Helper.test? ? 1 : 10
```

Comment:
```suggestion
    NUMBER_OF_THREADS = Helper.test? ? 1 : [ENV.fetch("DELIVER_NUMBER_OF_THREADS", 10).to_i, 10].min
```

This is a little gross but the idea is here we default and max to 10 threads but allow the user to change to 1 to 9. This allows for instant fixing if rate limiting or machine can't handle 🤷‍♂️